Packs/Core/Playbooks/playbook-Scheduled_task_created_with_HTTP_or_FTP_reference_README.md

```diff
@@ -0,0 +1,55 @@
+This playbook is designed to handle the alert "Scheduled task created with HTTP or FTP reference".
+
+The playbook executes the following stages:
+
+Investigation:
+During the alert investigation, the playbook will perform the following:
+- Checks the IP and the URL reputation.
+- Checks the CGO process signature.
+- Search for related XDR agent alerts to determine if the creation of the scheduled task is part of an attack pattern.
```

Comment:
Searches for related XDR agent alerts to determine if the creation of the scheduled task is part of an attack pattern.
Comment